packages/foreman/rubygem-mail/rubygem-mail.spec

```diff
@@ -31,6 +34,12 @@ Documentation for %{name}.
 %prep
 %setup -q -n  %{gem_name}-%{version}
 
+# On EL8 rubygem-net-smtp/imap/pop are bundled into ruby-libs package and
```

Comment:
Just leaving it as a note here that if anyone ever gets back to this: technically it's not just EL8, but also anything else that packages Ruby < 3. Not that we support those, so close enough. We need to deal with this prior to EL9 anyway since there is no Ruby 2.7 there.